docs/uninstall-olm.md

```diff
@@ -0,0 +1,53 @@
+# How do I uninstall OLM?
+
+You can uninstall OLM in a symmetrical way as you did to install it. 
+
+## Uninstall Released OLM
+For uninstalling release versions of OLM, for example version 0.13.0, you can use the following command:
+
+```bash
+export olm_release=0.13.0
+kubectl delete -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_release}/crds.yaml
+kubectl delete -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_release}/olm.yaml
+```
+
+> NOTE: You can identify which version of OLM you are using by inspecting the version of the packageserver CSV.
+
+> ```bash
+> kubectl -n <OLM namespace, e.g.: olm, etc> get csvs
+> NAME DISPLAY VERSION REPLACES PHASE
+> packageserver Package Server 0.13.0 Succeeded
+> ```
+
+## Uninstall From Git Repository Master Branch
+
+You can also uninstall OLM from the master branch of the [operator-framework/operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager/) repository with the following: 
+
+```bash
+kubectl delete -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/crds.yaml
+kubectl delete -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/olm.yaml
+```
+If you have cloned the git repository and use the [Makefile](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Makefile) for deploying OLM locally for development purposes, you can remove OLM using the following commands:
+
+```bash
+cd operator-lifecycle-manager
+kubectl delete -f ./deploy/upstream/quickstart/crds.yaml
+kubectl delete -f ./deploy/upstream/quickstart/crds.yaml
+```
+
+## Verify OLM Uninstall
+
+You can verify that OLM has been uninstalled successfully by making sure that the necessary CustomResourceDefinitions are removed:
```

Comment:
`the necessary` should be replaced with `OLM **owned** CustomResourceDefinitions`